src/Microsoft.Performance.SDK/Processing/IApplicationEnvironmentV2.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Microsoft.Performance.SDK.Auth;
+
+namespace Microsoft.Performance.SDK.Processing
+{
+    /// <summary>
+    ///     Extends <see cref="IApplicationEnvironment"/> to provide additional functionality.
+    /// </summary>
+    [Obsolete("This interface will be removed in version 2.0 of the SDK. It is OK to use this interface in version 1.x of the SDK.")]
+    public interface IApplicationEnvironmentV2
```

Comment:
are we using a 'V' for version in interfaces? (instead of 'IApplicationEnvironment2')?

Reply:
not sure why this was duplicated